### ISSUE TITLE
help copy on committee detail pages

### ISSUE THREAD

**Issue by davidmooreppf** [datamade/nyc-councilmatic#91]
Suggestion for help text on top of committee detail pages, e.g. [Committee on Aging](http://nyc-councilmatic.datamade.us/committee/committee-on-aging/). I often get too wordy, so can be overruled. 

Thinking of this as a short-term solution to making committee pages more informative at first glance, easier than bringing over the full [List view](http://legistar.council.nyc.gov/DepartmentDetail.aspx?ID=6905&GUID=0D8F5FED-57D4-42FA-AA98-0F26D09E7158&Search=) of Committee Activity. 

Suggest an info box up top, with copy : 

How can you stay informed about this committee's activity? We're developing alert features, so you can be notified when a new public event such as a committee hearing is scheduled. Then, you'll be able to send your feedback to committee members on the official agenda, transcript, and meeting minutes. But for now, visit the calendar view on the official committee website to see past and upcoming events.

... end links off, as in #65, to legistar page.

**Comment by davidmooreppf** [comment]
I confess I think this is useful on top of each committee detail page, but it a version could also live on top of Events. But I realize I'm pushing it, with the wordiness & feature descriptions.

**Comment by derekeder** [comment]
@davidmooreppf I think we should make this content more about explaining what is on the page rather than the features we plan on adding.

For the [events page](http://nyc.councilmatic.org/events/), we used this:

 Most of the Council's legislative action happens in committees. Each committee meets in public hearings 2-3 times per month where proposed legislation is debated. Members of other government branches, as well as the public, are able to attend and comment at these hearings.

Meetings of the entire Council, referred to as Stated Meetings, occur twice a month at City Hall. 

... and link to the about page.

**Comment by davidmooreppf** [comment]
That makes sense. We'll focus on beefing up "how council works" explainers.
